installing_on_kubernetes/_topics/configure_oidc.md

```diff
@@ -0,0 +1,19 @@
+## Configuring OpenID-Connect Authentication
+
+To run with OpenID-Connect Authentication, customize the following example as required to fit your environment.
+For this example we tested with Keycloak version 11.0
+
+Create a secret containing the OpenID-Connect's `Client ID` and `Client Secret`.
+The values for `CLIENT_ID` and `CLIENT_SECRET` come from your authentication provider's client definition.
+
+```bash
+$ oc create secret generic <name of your kubernetes secret> --from-literal=CLIENT_ID=<your auth provider's client ID> --from-literal=CLIENT_SECRET=<your auth provider's client secret>
```

Comment:
Ugh, sorry, I didn't realize the single quote would cause the bash syntax highlight to be annoying - what about on the following:

```suggestion
$ oc create secret generic <name of your kubernetes secret> --from-literal=CLIENT_ID=<your auth provider client ID> --from-literal=CLIENT_SECRET=<your auth provider client secret>
```

or

```suggestion
$ oc create secret generic <name of your kubernetes secret> --from-literal=CLIENT_ID=<your auth provider\'s client ID> --from-literal=CLIENT_SECRET=<your auth provider\'s client secret>
```

Reply:
This would apply to the other places we did a quote in the placeholder as well.